modules/apps/31-ibc-query/client/cli/cli.go

```diff
@@ -1 +1,39 @@
 package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd returns the query commands for 31-ibc-query
+func GetQueryCmd() *cobra.Command {
+	queryCmd := &cobra.Command{
+		Use:                        "ibc-query",
+		Short:                      "IBC query command",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+	}
+
+	queryCmd.AddCommand(
+
```

Comment:
Need to implement queries.
I think query for getting submitted cross chain query is good.